test/EFCore.MySql.IntegrationTests/README.md

```diff
@@ -4,23 +4,25 @@ Integration and Performance Tests
 **Configuring the Database**
 
 1. Configure your MySQL database by opening the `config.json.example` file, specifying the connection string and saving the changed file as `config.json`.
-2. Run the `scripts/rebuild.sh` script on Linux or the `scripts/rebuild.ps1` script on Windows to rebuild all migrations. Any time you make changes to database models, run the rebuild script again.
+2. Run the `scripts/rebuild.ps1` script on Linux* or Windows to rebuild all migrations. Any time you make changes to the database models, run the rebuild script again.
+
+PS* - To install powershell on Linux read [this](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-linux) .
 
 **Running Integration Tests**
 
-1. Ensure that you configured the database (see previous paragraph).
+1. Ensure that you configured the database correctly (see previous paragraph).
 2. Run `dotnet test`. This will execute all tests in the Tests/ directory.
 
 **Running Performance Tests**
 
-1. Configure the Database
+1. Configure the database.
 2. Run `dotnet run`. This will start a .NET Core MVC API application on "http://localhost:5000".
 
 Methods:
 
-`GET  /api/async` and `GET /api/sync` return the most recent 10 posts.
+`GET  /api/async` and `GET /api/sync` returns the most recent 10 posts.
 
-`POST /api/async` and `POST /api/sync` create a new post. The request body should be `Content-Type: application/json` in the form:
+`POST /api/async` and `POST /api/sync` creates a new post. The request body should be `Content-Type: application/json` in the form:
```

Comment:
Should this really be `returns`/`creates` (3rd person singular) instead of `return`/`create` (3rd person plural)? While I am not a native speaker, I would think that an enumeration of two things should be considered plural as in _they return_ (and it _sound_ weird to me using `returns`/`creates` here as well).

Reply:
> I am not a native speaker as well, but English is more or less my second language as all our education was more or less in it :) . In both the cases i.e. for return and create, they are an action being done which is a verb. It is singular when it is in a form of a question or when you ask some one like - Can you return my radio?. When someone returns the radio, it becomes like - Abc returns the radio - or Abc returned the radio ( past tense) - although it is only one radio. We referred Wren and Martin for English grammar in our school. You can refer it too? if that helps, there is plenty of pdf's online for it.

@ragavendra I am aware of general English grammar. In your example, a _single_ entity returns the radio: `Alice returns the radio.`

That is not the case in the lines that you changed. Those contain _two_ entities. Your example would then become `Alice and Bob returns the radio.` That is wrong, since Alice _and_ Bob are 3rd person plural, not singular. It should be `Alice and Bob return the radio.`

So I still believe it was correct before, and the change should be reverted. If you still think I am wrong about it, please reference a couple of similar sentences from some official docs.

(There are language experts on the .NET docs team. Feel free to ping one, if you feel strongly about it and if they want to take a look at it.)

Reply:
Never mind @lauxjpn . For now I have updated it like before. If I find relevant sentences, I will update :) .